airflow/providers/fab/provider.yaml

```diff
@@ -36,6 +36,11 @@ versions:
 dependencies:
   - apache-airflow>=2.9.0
   - flask>=2.2,<2.3
+  # We are tightly coupled with FAB version as we vendored-in part of FAB code related to security manager
+  # This is done as part of preparation to removing FAB as dependency, but we are not ready for it yet
+  # Every time we update FAB version here, please make sure that you review the classes and models in
+  # `airflow/providers/fan/auth_manager/security_manager/override.py` with their upstream counterparts.
```

Comment:
typo
```suggestion
  # `airflow/providers/fab/auth_manager/security_manager/override.py` with their upstream counterparts.
```

Reply:
Maybe this note does not belong to this PR and I lag a bit of history and when changing/fixing something in FAB last time I was wondering why actually we have a partly fork of the same code - I assume this is for historic reasons. I'd rather favor to really clearly separate the files or contribute upstream the diff... but this would be also outside of this PR actually...

Reply:
There are good reasons (historical) of course. And the reason is that originally FAB security manager does a bit too much - namely it also exposes FAB-version of user management (which we implemented on our own and this is what is available in Airflow UI.  But yeah. That's a topic for a different discussion. And actually with FAB provider separated, it **could** be possible to get rid of that partial copying - though it might be rather complex thing to do.

BTW. I was not the one who decided on it, but if  I was around that I'd also strongly oppose that decision and try to find another solution - what we do now is borderline crazy.

Reply:
Typo is still there :)

Reply:
Fixed.